pom.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>edu.tamu</groupId>
   <artifactId>cap</artifactId>
   <version>0.6.3-SNAPSHOT</version>
-  <packaging>war</packaging>
+  <packaging>${packaging}</packaging>
```

Comment:
I request doing the following to be consistent.
Change packaging in this manner:
```
  <properties>
    <maven.packaging>war</maven.packaging>
  </properties>

  <packaging>${maven.packaging}</packaging>

  <profiles>
    <profile>
      <id>war</id>
      <properties>
        <maven.packaging>war</maven.packaging>
      </properties>
    </profile>
    <profile>
      <id>jar</id>
      <properties>
        <maven.packaging>jar</maven.packaging>
      </properties>
    </profile>
  </profiles>
  ```

Reply:
You can also do this:
```
  <properties>
    <maven.packaging>${packaging}</maven.packaging>
  </properties>
  ```
  Preserving the behavior of using `${packagin}` by default.